test/benchmarks/src/main/scala/scala/collection/mutable/ArrayDequeBenchmark.scala

```diff
@@ -0,0 +1,147 @@
+package scala.collection
+package mutable
+
+import java.util.concurrent.TimeUnit
+
+import scala.util.Random
+
+import org.openjdk.jmh.annotations._
+import org.openjdk.jmh.infra.Blackhole
+
+@BenchmarkMode(Array(Mode.AverageTime))
+@Fork(2)
+@Threads(1)
+@Warmup(iterations = 10)
+@Measurement(iterations = 10)
+@OutputTimeUnit(TimeUnit.MICROSECONDS)
+@State(Scope.Benchmark)
+class ArrayDequeBenchmark {
+
+  @Param(Array("0", "1000"))
```

Comment:
maybe "100", "1000" instead?

Reply:
The benchmark should be able to run with a zero-length array to start, IMO.  I'll add as a comment, so that's clear.